heat_and_trees/heat_and_trees.ipynb

```diff
@@ -619,38 +649,49 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# url = 'http://data.phl.opendata.arcgis.com/datasets/957f032f9c874327a1ad800abd887d17_0.geojson'\n",
-    "url = 'https://opendata.arcgis.com/api/v3/datasets/5abe042f2927486891c049cf064338cb_0/downloads/data?format=geojson&spatialRefId=4326&where=1%3D1'\n",
-    "trees_gdf = gpd.read_file(url)\n",
+    "trees_gdf = gpd.read_file('data/PPR_Tree_Inventory_2022.geojson')\n",
     "trees_gdf.head()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The trees plot can be generated straight from the geopandas dataframe, but that is rather slow. By inspecting the dataframe, we can see that each tree is represented as a point. We can create a simpler pandas dataframe with lat as one column and lon as the other."
+    "The trees plot can be generated straight from the geopandas dataframe, but that is rather slow. By inspecting the dataframe, we can see that each tree is represented as a point. We can create a simpler pandas dataframe with `lat` as one column and `lon` as the other."
```

Comment:
Could you please check if it is still slow to plot with the geopandas dataframe directly? And if it's slow, how much slower is it compared to with Pandas directly?

Reply:
Actually it's no longer slow. So maybe replace that whole section with the `trees_gdf` plot directly?